### ISSUE TITLE
[Media] Windows Phone RT 8.1 - Bug in the TakePhotoAsync() method

### ISSUE THREAD

**Issue by jamesmontemagno** [jamesmontemagno/MediaPlugin#5]
_From @avorobjovs on June 3, 2016 13:52_

Please take a moment to fill out the following (change to preview to check or place x in []) and remove all unused areas

This is a 
- [x] Bug
- [] Feature Request

Which plugin does this impact:
- [] Battery
- [] Connectivity
- [] Contacts
- [] DeviceInfo
- [] ExternalMaps
- [] Geolocator
- [x] Media
- [] Permissions
- [] Settings
- [] Text To Speech
- [] Vibrate
- Other:

## Bug

Version Number of Plugin: 2.3.0
Device Tested On: Nokia Lumia 830, Windows Phone 8.1 Update 2, OS version 8.10.15148.160
Simulator Tested On:

### Expected Behavior

The TakePhotoAsync() method returns MediaFile object.

### Actual Behavior

The TakePhotoAsync() method raises an exception.

### Steps to reproduce the Behavior

1. Create new Windows Phone RT 8.1 application. 
2. Add a button to the MainPage.xaml
3. Add an event handler method for the button to the MainPage.xaml.cs code-behaviour file,
4. In the event handler method call the CrossMedia.Current.Initialize() method, then call the CrossMedia.Current.TakePhotoAsync() method.
5. The "Windows.UI.Xaml.Markup.XamlParseException: XAML parsing failed." exception is thrown.

=======================

Full exception stack trace:

Windows.UI.Xaml.Markup.XamlParseException: XAML parsing failed.
at Windows.UI.Xaml.Application.LoadComponent(Object component, Uri resourceLocator, ComponentResourceLocation componentResourceLocation)
at DMX.Helper.CameraCaptureUI.InitializeComponent()
at DMX.Helper.CameraCaptureUI..ctor()
at Plugin.Media.MediaImplementation.d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
at Media.WP.MainPage.d__11.MoveNext()

========================

There is a small test project in the attachment to check and reproduce this bug.

[Media.WP.zip](https://github.com/jamesmontemagno/Xamarin.Plugins/files/297744/Media.WP.zip)

_Copied from original issue: jamesmontemagno/Xamarin.Plugins#321_

**Comment by jamesmontemagno** [comment]
_From @abhimanyusinghal on June 6, 2016 10:20_

We are also getting same exception in our project

**Comment by jamesmontemagno** [comment]
_From @Thowk on June 7, 2016 7:38_

Same issue here. The problem is caused by project configuration. Xaml files are missing in output library.
At least, some workaround exist (simply manually compile that project and reference):
http://forums.xamarin.com/discussion/comment/201503/#Comment_201503

**Comment by letsar** [comment]
Hi,

We had the something similar in one of our project.
What we did was to add the .pri, .xr.xml and the xbf files into the nuget package. Have you tried this?
Sample of what we have:

``` xml
<file src="MyPlugin.W81\bin\Release\**\*.xbf" target="lib\NetCore45\MyPlugin\" />
<file src="MyPlugin.W81\bin\Release\MyPlugin.xml" target="lib\NetCore45\MyPlugin.xml" />
<file src="MyPlugin.W81\bin\Release\MyPlugin.pri" target="lib\NetCore45\MyPlugin.pri" />
<file src="MyPlugin.W81\bin\Release\MyPlugin.xr.xml" target="lib\NetCore45\MyPlugin\MyPlugin.xr.xml" />
```

**Comment by jamesmontemagno** [comment]
2.5.0-beta5 has been fixed.

**Comment by jamesmontemagno** [comment]
Fixed officially in 2.5.1-beta33! Tested and shipped!
